packages/middleware-joi/package.json

```diff
@@ -0,0 +1,49 @@
+{
+  "name": "@marblejs/middleware-joi",
+  "version": "0.2.2",
+  "description": "A joi validation middleware for Marble.js",
+  "main": "dist/index.js",
+  "types": "./dist/index.d.ts",
+  "scripts": {
+    "start": "npm run watch",
+    "watch": "tsc -w",
+    "build": "tsc",
+    "test": "jest --config ../../jest.config.js",
+    "prepack": "npm run build",
+    "prepare": "npm run build"
+  },
+  "files": [
+    "dist/"
+  ],
+  "keywords": [
+    "marble.js",
+    "joi",
+    "middleware",
+    "http",
+    "rxjs"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/marblejs/marble.git"
+  },
+  "engines": {
+    "node": ">= 8.0.0",
+    "npm": ">= 5.0.0"
+  },
+  "author": "Lúcio Rubens <luciorubeens@gmail.com>",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/marblejs/marble/issues"
+  },
+  "peerDependencies": {
+    "rxjs": "~6.1.0"
+  },
+  "homepage": "https://github.com/marblejs/marble#readme",
+  "dependencies": {
+    "@marblejs/core": "^0.2.2",
```

Comment:
Update after recent repository changes:

Add `@marblejs/core` as a `peerDependency` and `devDependency` (see: https://github.com/marblejs/marble/blob/master/packages/middleware-logger/package.json). It will allow `lerna` to link local library version instead of npm version + it will optimize end user dependency tree after `npm install`.